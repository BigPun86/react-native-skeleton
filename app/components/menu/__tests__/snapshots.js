import {Text} from 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../index.js';

it('renders correctly without props', () => {
  const tree = renderer.create(<Component/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with children', () => {
  const view = (
    <Component>
      <Text>Dashboard</Text>
      <Text>About</Text>
    </Component>
  );
  const tree = renderer.create(view).toJSON();
  expect(tree).toMatchSnapshot();
});
