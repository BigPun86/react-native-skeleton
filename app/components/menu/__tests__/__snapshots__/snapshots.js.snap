exports[`test renders correctly with children 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#444",
      "flex": 1,
      "paddingTop": 20,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Dashboard
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    About
  </Text>
</View>
`;

exports[`test renders correctly without props 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#444",
      "flex": 1,
      "paddingTop": 20,
    }
  } />
`;
